Ignore chrome devtools requests for Automatic Workspace Folders

cbliard opened the pull request:
Chrome devtools sends requests to `/.well-known/appspecific/com.chrome.devtools.json`, and when debugging feature tests with the devtools opened in Chrome, it makes the test fail.

Ignore these requests in test environment by replying 404 for a better Developer eXperience..

myabc commented:
Why not just add an empty JSON file to `/public`?

myabc commented:
(we already have a `public/.well-known` folder with metadata to ensure mobile app deep-links work)

myabc reviewed (changes requested):
see other comment.

cbliard commented:
> Why not just add an empty JSON file to `/public`?

Because I did the simplest thing that could possibly work while debugging a failing feature test. Please feel free to amend this PR with your suggestions @myabc.

And make sure that it still "works" with an empty JSON file: running feature test `./spec/features/work_packages/details/relations/hierarchy_spec.rb:151` with `OPENPROJECT_TESTING_NO_HEADLESS=1 OPENPROJECT_TESTING_AUTO_DEVTOOLS=1` on CLI should work.

myabc committed:
Handle chrome devtools requests for Automatic Workspace Folders

Chrome devtools sends requests to
`/.well-known/appspecific/com.chrome.devtools.json`, and when debugging
feature tests with the devtools opened in Chrome, it makes the test
fail.

For now we will serve a empty static file.

myabc commented:
We should revisit this once the esbuild work (#19132) is merged - as we may be able to take advantage of [vite-plugin-devtools-json](https://github.com/ChromeDevTools/vite-plugin-devtools-json).

cbliard commented:
Thanks for the simpler approach. Please approve and merge as I can't do it myself.
